astropy/modeling/optimizers.py

```diff
@@ -216,11 +216,10 @@ def __call__(self, objfunc, initval, fargs, **kwargs):
         """
         if 'maxiter' not in kwargs:
             kwargs['maxiter'] = self._maxiter
-        if 'acc' not in kwargs:
-            kwargs['acc'] = self._acc
+  
```

Comment:
Wouldn't this be a problem in case someone passed `xtol` in `kwargs` since self._acc is already passed in as `xtol`.

The idea was to have a uniform interface to all fitters (as much as possible) by using the same keyword names. In the case of `acc` it is passed to `xtol` in Simplex and LevMar but after checking the SLSQP code it looks like it has a meaning of `ftol` in SLSQP. I wonder if this approach is actually not helping but creating a confusion and if it's better to expose the optimizer's keywords as they are in scipy. 

So if `acc` is to be passed as `xtol`, keep the above lines but pop out `acc`.